src/test/playlisttest.cpp

```diff
@@ -56,3 +57,52 @@ TEST_F(PlaylistTest, Relative) {
     EXPECT_EQ(QString("base/folder/../../bar.mp3"),
             parser.playlistEntryToFilePath("../../bar.mp3", "base/folder"));
 }
+
+TEST_F(PlaylistTest, m3uEndOfLine) {
+    QTemporaryFile m3uFile;
+    ASSERT_TRUE(m3uFile.open());
+    m3uFile.write("crlf.mp3\r\n");
+    m3uFile.write("cr.mp3\r");
+    m3uFile.write("lf.mp3\n");
+    // Check for Windows-1250 Euro Sign
+    m3uFile.write("EuroSign\x80.mp3\n");
+    m3uFile.write("end.mp3");
+    m3uFile.close();
+
+    QList<QString> entries = ParserM3u().parseAllLocations(m3uFile.fileName());
```

Comment:
```suggestion
    const QList<QString> entries = ParserM3u().parseAllLocations(m3uFile.fileName());
```

Reply:
I really wish there was a compiler warning for improving const correctness, but according the only stackoverflow answer I found, that seems do be deemed useless by the majority of C++ devs...